package.json

```diff
@@ -2,6 +2,9 @@
   "name": "lucid-ui",
   "version": "2.36.5",
   "description": "A UI component library from AppNexus.",
+  "engines": {
+    "node": ">=8.12.0"
```

Comment:
I think I need to remove this since it throws an error during npm install if the node engine doesn't match.  Is there a better way to flag that development must be done >=8, but it doesn't matter for consumers of the library?

Reply:
Good question. I've seen folks use a little bash script to check the version of node instead of `engines` but it's certainly not as clean. As far as I can tell there isn't a clear hook we could use to only run a script when running `yarn install` from within lucid (i.e. you're a developer working on lucid). Maybe we should just skip this problem for now and instead document it in the README?